financial-services-accelerator/components/org.wso2.financial.services.accelerator.event.notifications.service/src/main/java/org/wso2/financial/services/accelerator/event/notifications/service/util/EventNotificationServiceUtil.java

```diff
@@ -0,0 +1,156 @@
+ */
+
+package org.wso2.financial.services.accelerator.event.notifications.service.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wso2.carbon.identity.application.common.model.ServiceProvider;
+import org.wso2.carbon.identity.oauth2.IdentityOAuth2Exception;
+import org.wso2.carbon.identity.oauth2.util.OAuth2Util;
+import org.wso2.financial.services.accelerator.common.config.FinancialServicesConfigParser;
+import org.wso2.financial.services.accelerator.common.util.FinancialServicesUtils;
+import org.wso2.financial.services.accelerator.common.util.Generated;
+import org.wso2.financial.services.accelerator.consent.mgt.service.impl.ConsentCoreServiceImpl;
+import org.wso2.financial.services.accelerator.event.notifications.service.EventNotificationGenerator;
+import org.wso2.financial.services.accelerator.event.notifications.service.constants.EventNotificationConstants;
+import org.wso2.financial.services.accelerator.event.notifications.service.exception.FSEventNotificationException;
+
+import java.util.Optional;
+
+/**
+ * Default event notification validations.
+ */
+public class EventNotificationServiceUtil {
+
+    private static final Log log = LogFactory.getLog(EventNotificationServiceUtil.class);
+    private static volatile ConsentCoreServiceImpl consentCoreService;
+
+    /**
+     * This method is used to send the polling generator as per config.
+     *
+     * @return EventNotificationGenerator
+     */
+    public static EventNotificationGenerator getEventNotificationGenerator() {
+
+        return (EventNotificationGenerator)
+                FinancialServicesUtils.getClassInstanceFromFQN(FinancialServicesConfigParser.getInstance()
+                        .getEventNotificationGenerator());
+    }
+
+    /**
+     * This method is used to send the default realtime event notification request generator.
+     *
+     * @return RealtimeEventNotificationRequestGenerator
+     */
+    //TODO
+//    public static RealtimeEventNotificationRequestGenerator getRealtimeEventNotificationRequestGenerator() {
+//
+//        RealtimeEventNotificationRequestGenerator realtimeEventNotificationRequestGenerator =
+//                (RealtimeEventNotificationRequestGenerator) FinancialServicesUtils
+//                        .getClassInstanceFromFQN(FinancialServicesConfigParser.getInstance().
+//                        getRealtimeEventNotificationRequestGenerator());
+//        return realtimeEventNotificationRequestGenerator;
+//    }
+
+    /**
+     * Method to modify event notification payload with custom eventValues.
+     *
+     * @param jsonNode Json Node to convert
+     * @return String eventNotificationPayload
+     */
+    public static String getCustomNotificationPayload(JsonNode jsonNode) {
+
+        return jsonNode.toString();
```

Comment:
Can't we do a .toString() at the caller without using a new method?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/159/commits/782acc3a382f5f5f2905b4f3d07b83526261b666